content/blog/tina-v-0.69.0.md

```diff
@@ -0,0 +1,161 @@
+---
+title: 'TinaCMS V0.69'
+date: '2022-07-21T04:00:00.000Z'
+last_edited: '2022-07-21T04:00:00.000Z'
+author: Logan Anderson
+---
+
+TinaCMS Version `0.69` brings some extensive improvements to Tina. Here is a short list of a few of the major features:
+
+1.  New unified client that can be used on the frontend and backend.
+2.  Data layer is enabled by default
+3.  Better support for read only tokens
+4.  Clearer and more concise way of working with TinaCMS in CI
+
+To update to the latest version run:
+
+```bash
+yarn add tinacms@latest
+yarn add --dev @tinacms/cli@latest
+```
+
+## New Unified client
+
+TinaCMS has always been missing a "unified way" to query and fetch content. Previously, there was \`StaticRequest\`, "Read only token client", "Experimental Generated client" and "just using fetch". Now all of these have been reduced to a single client that can be used on both the frontend and backend.
+
+There are only a couple of small changes to be made in order to update.
+
+### Updates in `.tina/schema.{ts,js}`
+
+Instead of passing an `apiURL` into `defineConfig`, now the clientId, branch and read only token (NEW) are all configured in the schema and the generated `client` must be passed to the schema. The local graphql api url is used if the `--local` flag is passed.
```

Comment:
```suggestion
Instead of passing an `apiURL` into `defineConfig`, now the clientId, branch and read only token (NEW) are all configured in the schema. The generated `client` must be passed to `defineConfig`.

Instead of needing to configure localhost as the `apiURL`, the local graphql api is now used by default with `yarn dev`. 
```
@logan-anderson, This look correct?

Reply:
Yup!